apps/web/src/pages/quick-start/components/demo-flow/nodes/DigestNode.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Handle, Position } from 'react-flow-renderer';
+
+import { NodeButton } from './NodeButton';
+import { DigestGradient } from '../../../../../design-system/icons/general/DigestGradient';
+
+export function DigestNode({ data, id }: { data: any; id: string }) {
```

Comment:
@LetItRock FYI
Created a separate component for every button so it will be easy to create separate logic for every button (like the trigger button on the playground etc..)
If we see the need to merge them into one generic component we most definitely can :)